design/one-pager-configuration-stacks.md

```diff
@@ -70,12 +70,10 @@ configurations and be able to manage those configuration sets on a higher level.
 
 It is important that the design puts forward:
 
-* A base boilerplate package where creating a new configuration set that has a
-  controller is as easy as changing the YAML files.
-* A way to declare pre-requisites that needs to be satisfied by the cluster
-  for the configuration set to work.
-* A way to change a high level configuration by changing 1 value that is then
-  translated to the YAMLs that are already deployed.
+* A way for infrastructure owners to write their configuration sets with
+  reconciliation easier than writing a controller from scratch.
```

Comment:
It feels like we're really close to having something that allows infrastructure owners to write their configuration sets without writing _any_ controller. Should that be a goal?